### PR TITLE
Added submission blob-arena

### PR DESCRIPTION
**Pull request by ben-jay-amin**

### CODE REVIEW

**Comment by GrugLikesRocks** [comment]
The Grugs are back for this edition of Dojo game jam bringing to you some on-chain degeneracy with our Blob Arena submission

https://github.com/GrugLikesRocks/Blob-arena

Quote from my Blobert: "When I snap these fingers, it’s not just drama I’m unleashing, it’s a whole spectacle. You ready?"

grugbert

File: submissions/blob-arena.md
Comment (GrugLikesRocks) [inline submissions/blob-arena.md:1]: Project Summary

"Blob arena" is a mini-game designed for a game jam, featuring unique characters known as "Bloberts" engaging in strategic, Pokémon-like battles. Players will navigate through exciting encounters, against other players or against AI, using their Bloberts’ distinctive traits to outsmart and defeat opponents. The game’s core mechanic revolves around an enhanced rock-paper-scissors style combat modified by each character’s attributes.

File: submissions/blob-arena.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Blob Arena
+
+### Project Summary
+Give a short summary of your game project.
+> Tell us about your game!
```
Comment (tarrencev) [inline submissions/blob-arena.md:5]:
```suggestion
> Tell us about your game!

"Blob arena" is a mini-game designed for a game jam, featuring unique characters known as "Bloberts" engaging in strategic, Pokémon-like battles. Players will navigate through exciting encounters, against other players or against AI, using their Bloberts’ distinctive traits to outsmart and defeat opponents. The game’s core mechanic revolves around an enhanced rock-paper-scissors style combat modified by each character’s attributes.
```

File: submissions/blob-arena.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+### GitHub
+Provide the GitHub where your project is hosted. Please ensure your github repo is registered with [OnlyDust](https://app.onlydust.com/p/create) to receive awards.
+> [Contracts](https://github.com/GrugLikesRocks/Blob-arena/)
+
+### Play
+Provide instructions on how to play your game. It could be a URL, a README, or a link to a binary to install.
+> [README.md](https://github.com/GrugLikesRocks/Blob-arena/edit/main/README.md)
```
Comment (tarrencev) [inline submissions/blob-arena.md:13]:
```suggestion
> [README.md](https://github.com/GrugLikesRocks/Blob-arena)
```

@ben-jay-amin @GrugLikesRocks the link to the game build in the repo readme looks dead. Can we upload the artifacts to github instead? You might need to use LFS

**Review by tarrencev — COMMENTED** [review]
Thanks for the submission! Looking forward to checking it out. Please fix the build url so we can easily play then we can merge the submission

**Pull request merged**